src/query/formats/src/field_encoder/row_based.rs

```diff
@@ -165,6 +167,20 @@ pub trait FieldEncoderRowBased {
         self.write_string_inner(s.as_bytes(), out_buf, raw);
     }
 
+    fn write_bitmap(
+        &self,
+        column: &StringColumn,
+        row_index: usize,
+        out_buf: &mut Vec<u8>,
+        raw: bool,
+    ) {
+        let v = unsafe { column.index_unchecked(row_index) };
+        let rb = RoaringTreemap::deserialize_from(v).unwrap();
+        let vals = rb.into_iter().collect::<Vec<_>>();
```

Comment:
this will write a very large string.

Reply:
if rb only contains one element, the output will still be very large.
```
rb.insert(10000000000);
````



I think binary is better.

Reply:
Sqlsmith needs to generate bitmap test values. How about adding a config item in `CommonSettings` to control whether to generate String or Binary values for Bitmap?